contract/r/gnoswap/gov/staker/api_history.gno

```diff
@@ -0,0 +1,65 @@
+package staker
+
+import (
+	"std"
+	"time"
+
+	"gno.land/p/demo/json"
+	"gno.land/p/demo/ufmt"
+)
+
+// GetPossibleVotingAddressWithWeight returns the max voting weight + and possible voting address with weight
+func GetPossibleVotingAddressWithWeight(endTimestamp uint64) (uint64, map[std.Address]uint64) {
```

Comment:
Possibly?

Reply:
ref: https://docs.gnoswap.io/core-concepts/governance#scope-of-proposals-5

When new proposal has been made, user can only vote with voting power  that has been already made 1 day before proposal creation time.
> So terms of `possible` has been used.

It's works like guard for manipulating proposal result by whale address.